.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Deploy docs
+on:
+  workflow_dispatch:

```

Comment:
Another request is to automate website updates. This is currently only performed manually.